LFPy/test/test_cell.py

```diff
@@ -51,7 +51,7 @@ def test_cell_tvec_00(self):
         tvec_numpy = np.linspace(0, stickParams['tstop'],
                     int(stickParams['tstop']/stickParams['dt']) + 1)
 
-        np.testing.assert_equal(tvec, tvec_numpy)
+        np.testing.assert_allclose(tvec, tvec_numpy, atol=1E-4)
```

Comment:
Pretty high tolerance. I tried `1E-9` in all these tests, even up to tstop=1000000 with dt=0.1, and they ran fine. Did you encounter any issues?